src/IntegratorConfig.php

```diff
@@ -66,6 +66,11 @@ class IntegratorConfig
      */
     protected const RECIPES_DIRECTORY = 'vendor/spryker-sdk/integrator/data/recipes/';
 
+    /**
+     * @var string
+     */
+    protected const LOCAL_RECIPES_DIRECTORY = 'vendor/spryker-sdk/integrator-recipes/';
```

Comment:
This must never be the case, we are not allowed to mess with vendor dir.
Use a local tmp/ path instead to work with, one that is excluded outside of project to avoid it accidently being commited (so also no /data etc).

Reply:
It can't be accidentally committed. You really do that on purpose.

Reply:
Either way, thats a hard rule we must follow.

Reply:
We discussed, will make follow up fixed